variables.md

```diff
@@ -495,6 +495,6 @@ Following variables are relevant for agama installation
 Variable        | Type      | Default value | Details
 ---             | ---       | ---           | ---
 AGAMA | boolean | 0 | Agama installation support
-AGAMA_AUTO | string | | The auto-installation is started by passing `agama.auto=<url>` on the kernel's command line
+AGAMA_AUTO | string | | The auto-installation is started by passing `inst.auto=<url>` on the kernel's command line
```

Comment:
INST_AUTO, INST_INSTALL_URL etc.

Reply:
right, better to align var name? Others LGTM. BTW this is new agama change, right? Usage changed?

Reply:
Hmm, thanks for your input, it makes sense, but sine the current settings are used by different QE squads
Let me fix it with a new ticket

Reply:
> right, better to align var name? Others LGTM. BTW this is new agama change, right? Usage changed?

Yes, please refer to https://agama-project.github.io/docs/user/boot_options